FE/todo-app/src/components/TopNavigation/TopNavigationUserCircle.tsx

```diff
@@ -19,7 +21,9 @@ export const UserCircle: FC = () => {
           <div className="border-b mx-10">
             <div className={`${styles.username} flex justify-center px-4 pb-2`}>{data.data?.username}</div>
           </div>
-          <button className={styles.logoutButton}>Logout</button>
+          <button className={styles.logoutButton} onClick={() => router.push("/login")}>
+            Logout
+          </button>
```

Comment:
Pokuď je jediný důvod proč button máš, pouze přesměrování, tak to prosím nědělej přes button ale použíl z nextu Link :)